### PR TITLE
✨Source Zendesk Support: New Stream - Audit Logs

### LINKED ISSUE #16868: Source Zendesk Support: new stream Audit Logs
## Tell us about the problem you're trying to solve
[Audit Logs](https://developer.zendesk.com/api-reference/ticketing/account-configuration/audit_logs/#content) show various changes in a Zendesk account since the account was created. Records of these changes are saved indefinitely and you can view the entire change history.

[Allowed for:](https://developer.zendesk.com/api-reference/help_center/help-center-api/articles/#allowed-for)
- Admins on accounts that have audit log access

**The Audit Logs endpoints are only available on the Enterprise plan.**

## Describe the solution you’d like
Research pros and cons for implementing this stream.
_Research should contain: what stream should be implemented, permissions and scopes (if needed), paid plan (if needed)._

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
Closes #16868

## Enhancement
New streams: 

- Audit Logs

## Recommended reading order
1. `source_stripe/source.py`
2. `source_stripe/streams.py`
3.  `source_stripe/schemas/*`

## 🚨 User Impact 🚨
No breaking changes

## Pre-merge Checklist
<details open><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter [Yes]

- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Issue acceptance criteria met
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
- [x] Documentation which references the generator is updated as needed
+

### CODE REVIEW

File: airbyte-integrations/connectors/source-zendesk-support/acceptance-test-config.yml [hunk 1]
```diff
@@ -5,37 +5,40 @@ test_strictness_level: high
...
+      - config_path: "secrets/config.json"
+        expect_records:
+          path: "integration_tests/expected_records.jsonl"
+          extra_fields: no
+          exact_order: no
+          extra_records: yes
+        fail_on_extra_columns: false
+        empty_streams:
+          - name: "audit_logs"
+            bypass_reason: "no records"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-zendesk-support/acceptance-test-config.yml:32]: @sajarin can you add a message saying this stream only works with Enterprise accounts? Also would be nice to some info in the documentation.

**Review by marcosmarxm — DISMISSED** [review]

**Review by sajarin — APPROVED** [review]

**Comment by sajarin** [comment]
@bazarnov could I get you to take a quick look at this and approve if you feel it is ready to be merged?

**Review by bazarnov — APPROVED** [review]
LGTM

**Pull request merged**